### ISSUE TITLE
Cloud Installation of the Public Results Application - build error @Heroku

### ISSUE THREAD

**Issue by marttimato** [jflamy/owlcms4#321]
https://jflamy.github.io/owlcms4/#/Remote -> Click 'Deploy to Heroku' causes build error:

-----> JVM Common app detected
-----> Installing JDK 1.8... done
-----> Run app detected
-----> Error: ./buildpack-run.sh is not executable.
       Make it executable by running 'chmod +x ./buildpack-run.sh'
       on your local machine, then commit and push as usual.
 !     Push rejected, failed to compile Run app.
 !     Push failed

**Comment by jflamy** [comment]
Somehow the git permission bits were lost. Restored them.  Will create a 4.5.2-rc7 prerelease to check
```
$ find . -name *.sh | xargs git update-index --chmod=+x
$ git ls-files --s | grep \\.sh
100755 c7ac89645a47defe76f81bfc12c4eeedefcf6f98 0       buildpack-run.sh
100755 c9326f570930fc9ec3a34dfd12f1aa4f3afcea44 0       src/main/filtered/buildpack-run.sh
```

**Comment by jflamy** [comment]
There were other things broken too.  Fixed in 4.5.2 stable release (or 4.5.2-rc11 which is the same)
